sig-node/wasmedge/support_wasmedge.md

```diff
@@ -0,0 +1,192 @@
+WasmEdge is a lightweight, high-performance, and extensible WebAssembly runtime for cloud native, edge, and decentralized applications. It powers serverless apps, embedded functions, microservices, smart contracts, and IoT devices. WasmEdge is currently a CNCF (Cloud Native Computing Foundation) Sandbox project.
```

Comment:
This is not consistent with the Chinese part. we'd better keep the both the same.

Reply:
> This is not consistent with the Chinese part. we'd better keep the both the same.

Ok, it has been modified